build-tools/config/webpack/webpack.partial.conf.plugins.js

```diff
@@ -30,6 +30,9 @@ exports.config = ({ config, isDevelopment, buildType, isPartials }) => webpackCo
     plugins.push(
       // enable HMR globally
       new webpack.HotModuleReplacementPlugin(),
+      new webpack.SourceMapDevToolPlugin({
+        filename: 'asset/[name].js.map',
```

Comment:
So setting a filename means it's outputted instead of inline, which makes the build process slower.

Personally I often have sourcemaps off in devtools, so I don't really like the inline-eval stuff - but it's good to be aware of this.

Reply:
So what do you recommend?

I made the update for isDevelopment, and now the `eval-cheap-module-source-map` is working as expected. So think the plugin is not necessary if it makes the build process slower.

I removed the plugin and tested again, and it worked as well with the inline-eval (`eval-cheap-module-source-map`).

I think we can move forward and revert this commit.

Reply:
> So think the plugin is not necessary if it makes the build process slower
both `devtool` and the plugin can be configured the same way, so it's more if we need the additional custom config - which in your version we don't :)

So with this, I would remove the plugin again.

What you could do, is test some of the different `devtool` settings, and see if they all work correctly (with `awesome-typescript-loader`). If that's the case, we're good after that. If not, we might still want to switch to `ts-loader`.